src/meta/meta_ckpt.c

```diff
@@ -932,13 +932,16 @@ __wt_meta_sysinfo_set(WT_SESSION_IMPL *session)
 {
     WT_DECL_ITEM(buf);
     WT_DECL_RET;
+    WT_TXN *txn;
     WT_TXN_GLOBAL *txn_global;
     wt_timestamp_t oldest_timestamp;
+    uint32_t i;
```

Comment:
Any better names like `snap_count` or something instead of just `i`, it is up to you.

Reply:
Updated the variable name.